src/sentry/migrations/0545_flat_file_size_cleanup.py

```diff
@@ -0,0 +1,97 @@
+    # schema of a table.
+    # Here are some things that make sense to mark as dangerous:
+    # - Large data migrations. Typically we want these to be run manually by ops so that they can
+    #   be monitored and not block the deploy for a long period of time while they run.
+    # - Adding indexes to large tables. Since this can take a long time, we'd generally prefer to
+    #   have ops run this and not block the deploy. Note that while adding an index is a schema
+    #   change, it's completely safe to run the operation after the code has deployed.
+    is_dangerous = False
+
+    dependencies = [
+        ("sentry", "0544_remove_groupsubscription_columns"),
+    ]
+
+    operations = [
+        # Add new `size` field
+        migrations.SeparateDatabaseAndState(
+            database_operations=[
+                migrations.RunSQL(
+                    """
+                    ALTER TABLE "sentry_artifactbundleflatfileindex" ADD COLUMN "size" integer NOT NULL DEFAULT 0;
+                    """,
+                    reverse_sql="""
+                    ALTER TABLE "sentry_artifactbundleflatfileindex" DROP COLUMN "size";
+                    """,
+                    hints={"tables": ["sentry_artifactbundleflatfileindex"]},
+                ),
+            ],
+            state_operations=[
+                migrations.AddField(
+                    model_name="artifactbundleflatfileindex",
+                    name="size",
+                    field=sentry.db.models.fields.bounded.BoundedPositiveIntegerField(default=0),
+                ),
+            ],
+        ),
+        # Give fields a default so they can be removed in followup migrations
+        migrations.AlterField(
+            model_name="artifactbundleindex",
+            name="release_name",
+            field=models.CharField(default="", max_length=250),
+        ),
+        migrations.AlterField(
+            model_name="debugidartifactbundle",
+            name="source_file_type",
+            field=models.IntegerField(default=2),
+        ),
+        # Remove already defaulted column from the state
+        migrations.SeparateDatabaseAndState(
+            database_operations=[],
+            state_operations=[
+                migrations.RemoveField(
+                    model_name="artifactbundleindex",
+                    name="date_last_modified",
+                ),
+                migrations.RemoveField(
+                    model_name="artifactbundleindex",
+                    name="dist_name",
+                ),
+            ],
+        ),
+        # TODO? Remove the foreign key constraint from the legacy `flat_file_index_id` column
+        # migrations.AlterField(
+        #     model_name="artifactbundleflatfileindex",
+        #     name="flat_file_index",
+        #     field=sentry.db.models.fields.foreignkey.FlexibleForeignKey(
+        #         db_constraint=False, on_delete=models.deletion.CASCADE, to="sentry.File"
+        #     ),
+        # ),
```

Comment:
Looks like I missed this step from the metadata migration in https://github.com/getsentry/sentry/pull/55764.
The docs only mention this step in the section about removing a whole table, so I’m unsure if it is strictly necessary, or if the docs need to be updated to highlight that it indeed is.

Reply:
Another confusing thing about this is that we still have a steady stream of queries for this, even though it has already been removed from the metadata a couple of days ago.

```
SELECT sentry_artifactbundleflatfileindex . id 
FROM sentry_artifactbundleflatfileindex 
WHERE sentry_artifactbundleflatfileindex . flat_file_index_id IN ( ? )
```

Reply:
That is really strange. Django shouldn't be querying fields that are not in the model. Could you be looking at older tracing data?